common/annotations/sr_Latn.xml

```diff
@@ -1704,305 +1704,305 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="🏔" type="tts">snežni vrh planine</annotation>
-		<annotation cp="⛰">planina</annotation>
+		<annotation cp="⛰">planina | planina</annotation>
 		<annotation cp="⛰" type="tts">planina</annotation>
-		<annotation cp="🌋">erupcija | planina | vulkan</annotation>
+		<annotation cp="🌋">erupcija | vulkan | planina</annotation>
 		<annotation cp="🌋" type="tts">vulkan</annotation>
-		<annotation cp="🗻">fudži | planina</annotation>
+		<annotation cp="🗻">planina | planina fudži | fudži</annotation>
 		<annotation cp="🗻" type="tts">planina fudži</annotation>
-		<annotation cp="🏕">kampovanje</annotation>
+		<annotation cp="🏕">kampovanje | kampovanje</annotation>
 		<annotation cp="🏕" type="tts">kampovanje</annotation>
 		<annotation cp="🏖">plaža | plaža sa suncobranom | suncobran</annotation>
 		<annotation cp="🏖" type="tts">plaža sa suncobranom</annotation>
-		<annotation cp="🏜">pustinja</annotation>
+		<annotation cp="🏜">pustinja | pustinja</annotation>
 		<annotation cp="🏜" type="tts">pustinja</annotation>
-		<annotation cp="🏝">ostrvo | pusto</annotation>
+		<annotation cp="🏝">ostrvo | pusto | pusto ostrvo</annotation>
 		<annotation cp="🏝" type="tts">pusto ostrvo</annotation>
 		<annotation cp="🏞">nacionalni park | park</annotation>
 		<annotation cp="🏞" type="tts">nacionalni park</annotation>
-		<annotation cp="🏟">stadion</annotation>
+		<annotation cp="🏟">stadion | stadion</annotation>
 		<annotation cp="🏟" type="tts">stadion</annotation>
-		<annotation cp="🏛">antička građevina | antika</annotation>
+		<annotation cp="🏛">antika | antička građevina</annotation>
 		<annotation cp="🏛" type="tts">antička građevina</annotation>
-		<annotation cp="🏗">gradilište | gradnja</annotation>
+		<annotation cp="🏗">gradnja | gradilište</annotation>
 		<annotation cp="🏗" type="tts">gradilište</annotation>
-		<annotation cp="🧱">cigla | cigle | glina | malter | zid</annotation>
+		<annotation cp="🧱">glina | zid | malter | cigla | cigle</annotation>
 		<annotation cp="🧱" type="tts">cigla</annotation>
-		<annotation cp="🪨">čvrsto | gromada | kamen | stena | teško</annotation>
+		<annotation cp="🪨">gromada | kamen | stena | teško | čvrsto</annotation>
 		<annotation cp="🪨" type="tts">kamen</annotation>
-		<annotation cp="🪵">cepanica | deblo | drvo | panj</annotation>
+		<annotation cp="🪵">deblo | drvo | panj | cepanica</annotation>
 		<annotation cp="🪵" type="tts">drvo</annotation>
 		<annotation cp="🛖">koliba | kuća | okrugla koliba</annotation>
 		<annotation cp="🛖" type="tts">koliba</annotation>
 		<annotation cp="🏘">kuće</annotation>
 		<annotation cp="🏘" type="tts">kuće</annotation>
-		<annotation cp="🏚">kuća | zapuštena kuća | zapušteno</annotation>
+		<annotation cp="🏚">zapušteno | zapuštena kuća | kuća</annotation>
 		<annotation cp="🏚" type="tts">zapuštena kuća</annotation>
-		<annotation cp="🏠">dom | kuća</annotation>
+		<annotation cp="🏠">dom | kuća | kuća</annotation>
 		<annotation cp="🏠" type="tts">kuća</annotation>
```

Comment:
Here there's a duplicate -- kuća appears twice
It seems that the old file was normalized (maybe by CLDRModify) and the new file is not normalized
This is a kind of regression; it might be fixed when CLDRModify is run again; it would save us some churn, though, if the normalization were combined with the algorithmic generation